chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+output=output
```

Comment:
Constants should be all caps with underscores, also consider adding `readonly` to make it clear.
```suggestion
#Constants
readonly OUTPUT=output
```